linux/base.Dockerfile

```diff
@@ -202,6 +202,30 @@ RUN wget -nv -O puppet-tools.deb https://apt.puppet.com/puppet-tools-release-bus
   && apt-get install puppet-bolt \
   && rm -f puppet-tools.deb
 
+# install Rust
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+  CARGO_HOME=/usr/local/cargo \
+  PATH=/usr/local/cargo/bin:$PATH
+
+RUN RUN set -eux; \
+  apt-get update; \
+  apt-get install -y --no-install-recommends \
+  ca-certificates \
+  gcc \
+  libc6-dev \
```

Comment:
I’m curious why this package is needed. Doesn’t the system come with libc on it?

Reply:
was trying to follow this guide, https://hub.docker.com/r/rustlang/rust/dockerfile @rylev